### PR TITLE
InputField with suffix select

### LINKED ISSUE #5633: [CAL-429] Input component - Suffix with dropdown
[Text Field Brief - ❖ Cal DS (Figma)](https://www.figma.com/file/9MOufQNLtdkpnDucmNX10R/%E2%9D%96-Cal-DS?node-id=346%3A30200&t=cfC1AJQvn652pNeu-1)

> From [SyncLinear.com](https://synclinear.com) | [CAL-429](https://linear.app/calcom/issue/CAL-429/input-component-suffix-with-dropdown)

### PR DESCRIPTION
**Pull request by sean-brydon**
God I hate react select

Fixes: #5633

CleanShot 2022-11-25 at 13 14 20@2x

### CODE REVIEW

**Review by emrysal — APPROVED** [review]
Approved, bonus points if you could implement on minimum booking notice 👍

**Review by zomars — APPROVED** [review]

**Comment by PeerRich** [comment]
> Approved, bonus points if you could implement on minimum booking notice 👍

@sean-brydon lets do that in a follow up PR

**Comment by sean-brydon** [comment]
> Approved, bonus points if you could implement on minimum booking notice 👍

Already on my list :)

File: packages/ui/index.tsx [hunk 1]
```diff
@@ -30,7 +31,7 @@ export { default as CheckboxField } from "./components/form/checkbox/Checkbox";
 export { default as ErrorBoundary } from "./ErrorBoundary";
 export { default as AddressInput } from "./form/AddressInputLazy";
 export { default as PhoneInput } from "./form/PhoneInputLazy";
-export { default as Select } from "./form/Select";
+export { default as Select, UnstyledSelect } from "./form/Select";
```
Comment (JeroenReumkens) [inline packages/ui/index.tsx:34]: Note that this is exporting from the OLD select component. I'm creating a PR where I'm deleting this line now, and exporting from `/components`.

**Pull request merged**